src/schemas/shexRs.ts

```diff
@@ -0,0 +1,196 @@
+export const ShExRSchema = `PREFIX sx: <http://www.w3.org/ns/shex#>
```

Comment:
I'm not sure what this is for - is this for defining a shexR shape? If so, we should try to host this somewhere instead of copying a shape locally.